README.md

```diff
@@ -44,6 +46,17 @@ https://github.com/customerio/customerio-ios.git
 
 ![](docs/img/xcode_spm_select_sdks.jpeg)
 
+### Available SDK packages
```

Comment:
🤔 What is the value of this section? 

Instead of us introducing each SDK from a technical standpoint, would it be better to instead introduce each *feature* of the SDK? In the docs for each feature, example FCM messaging, our docs can tell you "Make sure to install the FCM messaging SDK to use this feature...".

Reply:
Yeah, when I wrote this, I was questioning how deep we want this to go and did the bare minimum. Rather than "available sdk packages", it could be a feature support/package matrix—here's what we support and which package(s) you need to install to support the feature.